lib/TerminalDisplay.cpp

```diff
@@ -275,16 +275,10 @@ void TerminalDisplay::setVTFont(const QFont& f)
 {
   QFont font = f;
 
-  // This was originally set for OS X only:
-  //     mac uses floats for font width specification.
-  //     this ensures the same handling for all platforms
-  // but then there was revealed that various Linux distros
-  // have this problem too...
-  font.setStyleStrategy(QFont::ForceIntegerMetrics);
-
+  // Check if font is not fixed pitch and print a warning
   if ( !QFontInfo(font).fixedPitch() )
   {
-      qDebug() << "Using a variable-width font in the terminal.  This may cause performance degradation and display/alignment errors.";
+	  qDebug() << "Using a variable-width font in the terminal.  This may cause performance degradation and display/alignment errors.";
```

Comment:
Tab removed.  BTW:  I don't think spacing is 100% consistent in the file.

Reply:
> BTW: I don't think spacing is 100% consistent in the file.

Yes, the situation is less than ideal. Here I want to avoid to make it worse. Some more discussions can be found at https://github.com/lxqt/qtermwidget/issues/472